profiles/serializer/normalizer.py

```diff
@@ -17,34 +20,40 @@ def normalize_snippet(value: Any) -> Any:
 @normalize.register(Profile)
 def normalize_profile(profile: Profile) -> dict:
     data = normalize_profile_snippet(profile)
-    data['emailAddresses'] = [normalize(email) for email in profile.get_email_addresses()]
-    data['affiliations'] = [normalize(aff) for aff in profile.get_affiliations()]
+    data['emailAddresses'] = [normalize(email) for email in profile.get_email_addresses(True)]
+    data['affiliations'] = [normalize(aff) for aff in profile.get_affiliations(True, True)]
```

Comment:
We could get rid of the `include_restricted` option now as it's no longer needed?

Reply:
Depends on how the design of the access control ends up filtering the values, if it does it on `data` than the option is not needed but if it wants to push it down to `profile`...

Reply:
Err, yeah, need to hide it if not authorised. Long day...